arangod/Graph/Providers/ProviderTracer.cpp

```diff
@@ -132,6 +134,36 @@ void ProviderTracer<ProviderImpl>::addEdgeToBuilder(
   return _impl.addEdgeToBuilder(edge, builder);
 }
 
+template<class ProviderImpl>
+void ProviderTracer<ProviderImpl>::addEdgeIDToBuilder(
+    typename Step::Edge const& edge, arangodb::velocypack::Builder& builder) {
+  double start = TRI_microtime();
+  auto sg = arangodb::scopeGuard([&]() noexcept {
+    _stats["addEdgeToIDBuilder"].addTiming(TRI_microtime() - start);
+  });
```

Comment:
This is tracing.
I would like to measure this, just to avoid hitting an "unexpected" bottle neck.

Especially on a DBServer/SingleServer this method may do a SingleDocumentLookup to RocksDB, that can take significant amount of time (e.g. it is in the ballpark of adding the edge, if the edge itself is a smallish document)